freqtrade/templates/FreqaiExampleStrategy.py

```diff
@@ -182,7 +182,7 @@ def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame
         dataframe["sell_roi"] = dataframe["target_mean"] - dataframe["target_std"]
         return dataframe
 
-    def populate_buy_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+    def populate_entry_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
```

Comment:
I think you'll also want to update the column names from 'buy' and 'sell' to 'enter_long' and 'exit_long'
otherwise it's a mix between interface V2 and V3 - which may not play well.